src/components/VirtualizedGrid/__stories__/VirtualizedGrid.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import VirtualizedGrid from "../VirtualizedGrid";
+import Button from "../../Button/Button";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import { generateItems, itemRenderer } from "./virtualizedGrid.components";
+
+<Meta
+  title="Navigation/VirtualizedGrid"
+  component={ VirtualizedGrid }
+/>
+
+<!--- Component template -->
+
+export const virtualizedGridTemplate = createComponentTemplate(VirtualizedGrid);
+
+<!--- Component documentation -->
+
+# VirtualizedGrid
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Feedback](#feedback)
+
+## Overview
+VirtualizedGrid is a component which only renders visible grid items, it is a logic component and doesn't change and look and feel
+
+Under the hood we are using - [react-window](https://github.com/bvaughn/react-window) and [react-virtualized-auto-sizer](https://github.com/bvaughn/react-virtualized-auto-sizer)
+<Canvas>
+    <Story name="Overview" args={{}} >
```

Comment:
Please move this specific  story to be the return value of the variable virtualizedGridTemplate and pass the args parameter of the function as props to the VirtualizedGrid in the last row.
The reason for this structure is that only when we implement the overview story this way it's working with the controls mechanism of the storybook sandbox.
you can see an example in the mdx file of the AlertBanner component